cmd/skopeo/signing.go

```diff
@@ -4,35 +4,67 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"io"
 	"io/ioutil"
 
 	"github.com/containers/image/v5/signature"
-	"github.com/urfave/cli"
+	"github.com/spf13/cobra"
 )
 
 type standaloneSignOptions struct {
 	output string // Output file path
 }
 
-func standaloneSignCmd() cli.Command {
-	opts := standaloneSignOptions{}
-	return cli.Command{
-		Name:      "standalone-sign",
-		Usage:     "Create a signature using local files",
-		ArgsUsage: "MANIFEST DOCKER-REFERENCE KEY-FINGERPRINT",
-		Action:    commandAction(opts.run),
-		Flags: []cli.Flag{
-			cli.StringFlag{
-				Name:        "output, o",
-				Usage:       "output the signature to `SIGNATURE`",
-				Destination: &opts.output,
-			},
+func standaloneSignCommand(globalFlagResults *globalOptions) *cobra.Command {
+	var standaloneSignOpts standaloneSignOptions
+	standaloneSignCommand := &cobra.Command{
```

Comment:
The `standaloneSign` qualifiers in local variable names are such a large proportion of the text that it just makes telling the variables apart and reading the code difficult for no benefit.